### ISSUE TITLE
anti-PHEX antibody positivity

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#11601]
**Preferred term label:**
anti-PHEX antibody positivity

**Synonyms**
anti-phosphate regulating endopeptidase X-linked antibody positivity

**Definition (free text, please give PubMed ID)**
Acquired Osteomalacia Associated with Autoantibodies against PHEX
Published January 29, 2025
N Engl J Med 2025;392:513-515
DOI: 10.1056/NEJMc2405746

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Acquired Osteomalacia

**Your nano-attribution (ORCID)**
0000-0002-3387-1836
